lib/config.js

```diff
@@ -14,9 +14,10 @@
  * not, you can obtain one from Tidepool Project at tidepool.org.
  * == BSD2 LICENSE ==
  */
-var pkg = require('./../package.json');
+var pkg = require('../package.json');
 
-var config = (typeof window !== 'undefined') ? window.config : {};
+//var config = (typeof window !== 'undefined') ? window.config : {};
```

Comment:
Per below, I can see that there is some usefulness in differentiating outside pure functionality. We can use the `is-electron` npm package, which combines a pair of commonly used checks to see if the current runtime environment is electron or not.